cloud-sql/mysql/sqlalchemy/connect_connector.py

```diff
@@ -40,8 +40,9 @@ def connect_with_connector() -> sqlalchemy.engine.base.Engine:
     db_name = os.environ["DB_NAME"]  # e.g. 'my-database'
 
     ip_type = IPTypes.PRIVATE if os.environ.get("PRIVATE_IP") else IPTypes.PUBLIC
+    refresh_strategy = RefreshStrategy.LAZY
 
-    connector = Connector(ip_type)
+    connector = Connector(ip_type=ip_type, refresh_strategy=refresh_strategy)
```

Comment:
nit: there is no need to set `refresh_strategy` as it is only used in one place.

The Cloud SQL Python Connector also allows taking a `str` for `refresh_strategy` instead of the custom type.
If we use `"lazy"` it is a bit more user friendly for the sample as customers will not need to import the custom type of `RefreshStrategy` 😄 

Let's also add to the existing comment to explain why we are setting it. 

Something along the lines of "Setting 'refresh_strategy' argument to "lazy" enables lazy refresh mode which is recommended for serverless environments to avoid background refreshes from throttling CPU."
```suggestion
    connector = Connector(ip_type=ip_type, refresh_strategy="lazy")
```

Reply:
Same applies to other engine types.